lang/python/text-unidecode/Makefile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-text-unidecode
+PKG_VERSION:=1.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=text-unidecode-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://pypi.python.org/packages/source/t/text-unidecode/
```

Comment:
Hmm why pypi?

```
git grep pypi | wc -l
20
git grep pythonhosted | wc -l
74
```

Reply:
Let's use Pythonhosted, I don't mind to use it. I changed it in all my current PR.